rust/osm2lanes/src/locale.rs

```diff
@@ -171,6 +171,15 @@ impl std::str::FromStr for DrivingSide {
     }
 }
 
+impl ToString for DrivingSide {
+    fn to_string(&self) -> String {
+        match self {
+            Self::Right => String::from("right"),
```

Comment:
not sure if important,

but I avoided implementing this to avoid confusing right hand drive with right hand travel.

Reply:
I am confused about the difference, so maybe it is important! Is right hand drive when the driver gets in the right hand door of the car? I suppose this should be `Into<TagKey>` instead of `ToString` or something?